test/test_autograd.py

```diff
@@ -5546,7 +5546,7 @@ def fn(*inputs):
                             output_variable = getattr(self_variable, name)(*args_variable, **kwargs_variable)
                             if not isinstance(output_variable, tuple):
                                 output_variable = (output_variable,)
-                            inplace_self_variable = deepcopy(self_variable)
+                            inplace_self_variable = self_variable.detach().clone().requires_grad_(self_variable.requires_grad)
```

Comment:
Hmm, what's going on here?

Reply:
The deepcopy code seems to handle conj properly so this doesn't need to change?